test/bin/scenario.sh

```diff
@@ -533,6 +533,32 @@ EOF
         "$@"
 }
 
+# Public function to run the tests for an offline scenario.  Greenboot checks should be handled
+# in RF suite setup, using the qemu-guest-agent.py library.
+run_offline_tests(){
```

Comment:
We now have a way to switch off greenboot checks from the scenario. Do we still need this function?

Reply:
The comment could be better worded.  It's just that `run_tests` executes a greenboot check, but also that it expects a number of files contain network data to exist.  I'd opted to add the simpler function that doesn't include any of the network dependencies rather than nest most of the `run_test` func under a large `if` block.

Reply:
I would not want to hold the PR off because of this, but it's "easy" to detect the offline case by checking for an empty "ip" file, and, then just skipping the unnecessary settings. We do this in run scenario function. 
We can implement this enhancement in the next PR. Please resolve this comment.

Reply:
done